### PR TITLE
<Do Not Merge> Test mirror for slow pulls on f26-ah tester

### PR DESCRIPTION
**Pull request by yuqi-zhang**

### CODE REVIEW

File: .papr.sh [hunk 1]
```diff
@@ -41,9 +41,11 @@ if test -z "${INSIDE_CONTAINER:-}"; then
                -v $PWD:/code \
                -v /:/host \
                --workdir /code \
+               -v /etc/yum.repos.d:/etc/yum.repos.d.host:ro
```
Comment (jlebon) [inline .papr.sh]: Missing trailing backslash

File: .papr.sh [hunk 2]
```diff
@@ -41,9 +41,11 @@ if test -z "${INSIDE_CONTAINER:-}"; then
...
                -e INSIDE_CONTAINER=1 \
                -e PYTHON=$PYTHON \
-               registry.fedoraproject.org/fedora:26 /code/.papr.sh
+               registry.fedoraproject.org/fedora:26 /bin/sh -c
+               "cp -fv /etc/yum.repos.d{.host/*.repo,}h && /code/.papr.sh"
```
Comment (jlebon) [inline .papr.sh]: I think you have an errant `h` there. :)

**Comment by yuqi-zhang** [comment]
bot, retest this please

**Comment by giuseppe** [comment]
isn't this going to be a problem on CentOS?  I see we are pulling packages from the Fedora repos also on CentOS now

**Comment by yuqi-zhang** [comment]
Closing since issues are solved.

**Pull request closed without merging**